Dockerfile

```diff
@@ -14,8 +14,20 @@ RUN groupadd -g $userid -r vets-api && \
 RUN echo "deb http://ftp.debian.org/debian testing main contrib non-free" >> /etc/apt/sources.list
 RUN apt-get update
 RUN apt-get install -y -t testing poppler-utils
+# RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+#     dumb-init clamav clamdscan clamav-daemon imagemagick pdftk curl libpq5 vim libboost-all-dev
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    dumb-init clamav clamdscan clamav-daemon imagemagick pdftk curl libpq5 vim libboost-all-dev
+    dumb-init imagemagick pdftk curl libpq5 vim libboost-all-dev
+# Dependecies for ClamAV
```

Comment:
I am not fond of pulling in these tools into the base image. I tried to have them over into `development`, but fails when its on that side